packages/openneuro-app/src/scripts/search/filters-block-container.tsx

```diff
@@ -46,6 +60,19 @@ const FiltersBlockContainer: FC<FiltersBlockContainerProps> = ({
             Sentry.captureException(error)
           }
         }
+      } else if (!isModalityPath && param === "brain_initiative") {
+        removeFilterItem(setSearchParams)(param, value)
+        const queryParams = new URLSearchParams(location.search)
+        const query = queryParams.get("query")
```

Comment:
Similarly the hook useSearchParams() can return the query parameter so constructing a new URLSearchParams isn't needed.

Reply:
ah very nice

Reply:
had to redo a little after adding useSearchParams, but this update seems to be working as I expect it without constructing a new url params